packages/sdk/src/internal/subscriptionType.ts

```diff
@@ -7,4 +7,4 @@
  * A type representing the different parts of an Actor that can be subscribed to. When subscribed, the Actor will
  * receive updates when corresponding changes occur on the host.
  */
-export type SubscriptionType = 'transform' | 'rigidbody' | 'collider';
+export type SubscriptionType = 'transform' | 'rigidbody' | 'collider' | 'rigidbodyVelocity';
```

Comment:
>rigidbodyVelocity [](start = 73, length = 17)

Naming conventions for our string types are hyphen separated.  Should be `rigidbody-velocity I believe,